src/Manager.js

```diff
@@ -1,40 +1,34 @@
 class Manager {
   constructor(todaysDate, hotel) {
-    this.currentDate = todaysDate
+    this.currentDate = new Date(todaysDate)
     this.hotelRooms = hotel.allRooms
     this.todaysRevenue = 0;
-    this.availableRooms = [];
-    this.bookedRooms = [];
     this.todaysBookings = [];
     this.laterBookings = [];
   }
 
   getBookingStatus(bookingRepo) {
-    bookingRepo.forEach(booking => {
-        booking.bookingDate === this.currentDate ? this.todaysBookings.push(booking) : this.laterBookings.push(booking)
-    })
+    return bookingRepo.filter(booking => booking.bookingDate.getTime() === this.currentDate.getTime())
   }
 
   getBookedRooms(bookingRepo) {
-    this.hotelRooms.forEach(room => {
-      this.todaysBookings.forEach(booking => {
-        booking.roomNumber === room.number ? this.bookedRooms.push(room) : console.log();
-      })
-    })
+    let todaysBookings = this.getBookingStatus(bookingRepo)
+      return todaysBookings.map(booking => {
+        return this.hotelRooms.filter(room => room.number === booking.roomNumber)
+    }).flat()
```

Comment:
I like you use of .flat() array prototype here! That can come in super handy to make a single array out of nested arrays!